README.md

```diff
@@ -182,9 +182,52 @@ docker run -d --name jaeger -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 -e COLLECTOR_OTL
 1. Open [Jaeger in your browser](http://localhost:16686/)
 1. View the collected spans
 
-#### Logs
+### Logs
+
+This app using structured logging and for this purpose we use [Serilog](https://github.com/serilog/serilog-aspnetcore)
+
+For setting up Serilog you should call `AddSerilog` on [SerilogExtensions](TodoApi/Extensions/SerilogExtensions.cs) class and Add `Serilog` section with appropriate [Options](TodoApi/Extensions/SerilogExtensions.cs#L95)
+
+```json
+  "Serilog": {
+    "LogTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff} {Level} - {Message:lj}{NewLine}{Exception}",
+    "ElasticSearchUrl": "http://localhost:9200",
+    "SeqUrl": "http://localhost:5341",
+    "MinimumLevel": {
+      "Default": "Information",
+      "Override": {
+        "Microsoft": "Warning",
+        "Microsoft.Hosting.Lifetime": "Information"
+      }
+    }
+  }
```

Comment:
I'm not in love with this. Is this what everyone does for serilog configuration? I'd like to avoid anything custom if there's another way.